### ISSUE TITLE
Navigating App with a Screen Reader

### ISSUE THREAD

**Issue by windemissary** [Adventech/sabbath-school-lessons#4152]
Hi,
The effort you all put into this app is incredible and the resources available in place are very, very useful and comprehensive. Thank you.
I have a friend who is blind and uses the iOS screen reader that finds navigating the app difficult. 

For example, once they can not go to a previous day or ahead to the next day...for some reason swiping left or right while the screen reader is engaged does not work. The Ellen White Quotes are not select-able and so copy/paste doesn't work well. And overall navigating with a screen reader just seems more difficult from their perspective. 

I realize you all are very busy but I'm sure this effects a significant number of people. I'm not a programmer but I'm comfortable in a terminal and like to learn, so I'm willing to assist if that would be helpful. 

If you're interested in talking with my friend (directly or via email) let me know and I'll connect you. Thank you for your time and consideration. Grace to you.  Shawn.
